frontend/src/plugins/impl/TextInputPlugin.tsx

```diff
@@ -57,15 +63,26 @@ const TextComponent = (props: TextComponentProps) => {
     url: <GlobeIcon size={16} />,
   };
 
+  const endAdornment = props.maxLength ? (
+    <span className="text-muted-foreground text-xs font-medium">
```

Comment:
Maybe change to a more visible color (like an amber or red) when the max length is hit?

Reply:
i feel that is usually if you go over, instead of exactly, and this will just stop at the `max_length`

Reply:
maybe `amber` isn't as "woah something is wrong"

Reply:
OK, fine to keep as is for now. The only reason I suggested it is in case a user doesn't see the character count get confused why they can't continue typing